### PR TITLE
update readme to reflect website

### PR DESCRIPTION
**Pull request by geoffjentry**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,15 +73,18 @@ A [Workflow Management System](https://en.wikipedia.org/wiki/Workflow_management
...
+## Website and User Guide
+
+The [WDL website](https://software.broadinstitute.org/wdl/) is the best place to go for more information on both WDL and Cromwell. In particular new users should check out the [user guide](https://software.broadinstitute.org/wdl/userguide/) which has many tutorials, examples and other bits to get you started.
 
-## Mailing List
-The [Cromwell Mailing List](https://groups.google.com/a/broadinstitute.org/forum/?hl=en#!forum/cromwell) is cromwell@broadinstitute.org.
+## User Forum
 
-If you have any questions, suggestions or support issues please send them to this list. To subscribe you can either join via the link above or send an email to cromwell+subscribe@broadinstitute.org.
+If you have questions that aren't covered by the website you can ask them in the [user forum](http://gatkforums.broadinstitute.org/wdl/entry/signin?Target=categories%2Fask-the-wdl-team).
```
Comment (vdauwera) [inline README.md]: user forum -> support forum

http://gatkforums.broadinstitute.org/wdl/categories/ask-the-wdl-team

**Comment by vdauwera** [comment]
Minor edits, then :+1:

**Comment by knoblett** [comment]
:+1: for me.

**Comment by cjllanwarne** [comment]
No more mailing list?

File: README.md [hunk 2]
```diff
@@ -79,9 +79,9 @@ A [Workflow Management System](https://en.wikipedia.org/wiki/Workflow_management
 
 The [WDL website](https://software.broadinstitute.org/wdl/) is the best place to go for more information on both WDL and Cromwell. In particular new users should check out the [user guide](https://software.broadinstitute.org/wdl/userguide/) which has many tutorials, examples and other bits to get you started.
 
-## User Forum
+## Support Forum
 
-If you have questions that aren't covered by the website you can ask them in the [user forum](http://gatkforums.broadinstitute.org/wdl/entry/signin?Target=categories%2Fask-the-wdl-team).
+If you have questions that aren't covered by the website you can ask them in the [support forum](http://gatkforums.broadinstitute.org/wdl/entry/signin?Target=categories%2Fask-the-wdl-team).
```
Comment (vdauwera) [inline README.md]: Are you using the sign-in entry point on purpose? I would prefer to link directly to the category as some people (anyone coming in from GATK) will already be signed in.

**Comment by vdauwera** [comment]
Nope, no more.

File: README.md (hunk 2 above)
Comment (vdauwera) [inline README.md]: http://gatkforums.broadinstitute.org/wdl/categories/ask-the-wdl-team

Comment (geoffjentry) [inline README.md]: I pasted in the link that kate (sorry don't know her github) sent me - will switch

**Comment by geoffjentry** [comment]
@cjllanwarne we're not removing it but we're not advertising it eihter. if people come across it and send something that's fine

File: README.md (hunk 2 above)
Comment (geoffjentry) [inline README.md]: Oh i see what happened. when i go to that link i get the signin thing and i probably copy/pasted that into the readme

**Pull request merged**